src/middlewares/validarToken.ts

```
@@ -1,50 +1,31 @@
-import { jwtVerify } from "jose";
-import { type NextRequest, NextResponse } from "next/server";
+import { NextRequest, NextResponse } from "next/server";
+import { AuthenticationError, decodeToken, getEncodedSecret, getToken } from "@/utils/token";
+import { captureException } from "@sentry/nextjs";
 
 interface ExtendedNextRequest extends NextRequest {
 	user?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)